tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Remoting/Client/HttpProcessResponseExceptionIntegration.cs

```diff
@@ -41,9 +41,9 @@ public class HttpProcessResponseExceptionIntegration
         /// <returns>Calltarget state value</returns>
         internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, WebException webException, ref HttpWebResponse response)
         {
-            if (Tracer.Instance.InternalActiveScope is var scope)
+            if (Tracer.Instance.InternalActiveScope is { } scope)
```

Comment:
Not a fan of this kind of notation, because of its reduced readability, but I agree it is compact (and convenient)

Reply:
I'm a fan of pattern matching, but agree the syntax chosen for this particular pattern is obscure. Though I'd like to keep it as reduces the need for extra scope / nested ifs.